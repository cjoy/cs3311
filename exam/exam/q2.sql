-- COMP3311 12s1 Exam Q2
-- The Q2 view must have one attribute called (player,goals)

drop view if exists Q2;
create view Q2
as
... SQL code for view goes here ...
;
