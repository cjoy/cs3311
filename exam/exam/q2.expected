Christian Baumann|3
Johansson Henrik|2
Larsson Mikael|2
